[quant]ao migration for quantization mappings and fuser method mappings hg mv

HDCharles opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #64985
* #64982

moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

HDCharles committed:
[quant]ao migration for quantization mappings and fuser method mappings hg mv

moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant]ao migration for quantization mappings and fuser method mappings hg mv"


moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

[ghstack-poisoned]

supriyar commented on test/quantization/ao_migration/test_fuser_method_mappings.py:
```diff
@@ -0,0 +1,34 @@
+from torch.testing._internal.common_utils import TestCase
+
+
+class TestFuserMethodMappingsPyImport(TestCase):
```
I think so far all the tests have been put into the same file https://github.com/pytorch/pytorch/blob/master/test/quantization/ao_migration/test_quantize.py. We can probably rename the file name so it is more generic.

HDCharles replied:
ya thats been updated

supriyar commented on torch/quantization/fuser_method_mappings.py:
```diff
@@ -1,133 +1,15 @@
+This file is in the process of migration to `torch/ao/quantization`, and
+is kept here for compatibility while the migration process is ongoing.
+If you are adding a new entry/functionality, please, add it to the
+`torch/ao/quantization/MY_FILE.py`, while adding an import statement
```
replace MY_FILE with fuser_method_mapping?

HDCharles replied:
done

HDCharles committed:
Update on "[quant]ao migration for quantization mappings and fuser method mappings hg mv"


moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant]ao migration for quantization mappings and fuser method mappings hg mv"


moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant]ao migration for quantization mappings and fuser method mappings hg mv"


moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant]ao migration for quantization mappings and fuser method mappings hg mv"


moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant]ao migration for quantization mappings and fuser method mappings hg mv"


moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant]ao migration for quantization mappings and fuser method mappings hg mv"


moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant]ao migration for quantization mappings and fuser method mappings hg mv"


moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant]ao migration for quantization mappings and fuser method mappings hg mv"


moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

[ghstack-poisoned]

HDCharles committed:
Update on "[quant]ao migration for quantization mappings and fuser method mappings hg mv"


moving quantization_mappings.py and fuser_method_mappings.py to the ao folder while retaining backwards compatibility

Differential Revision: [D30930702](https://our.internmc.facebook.com/intern/diff/D30930702/)

[ghstack-poisoned]
